yang/ietf-l3vpn-ntw.yang

```diff
@@ -375,18 +375,25 @@ module ietf-l3vpn-ntw {
             container maximum-routes {
               description
                 "Defines 'maximum-routes' for the VRF.";
-              list address-family {
-                key "af";
+              list selector {
+                key "address-family protocol";
                 description
                   "List of address families.";
-                leaf af {
+                leaf address-family {
                   type identityref {
                     base vpn-common:address-family;
                   }
                   description
                     "Indicates the address family
                      (IPv4 or IPv6).";
                 }
+                leaf protocol {
+                  type identityref {
+                    base vpn-common:routing-protocol-type;
+                  }
+                  description
+                    "Indicates the routing protocol.";
```

Comment:
```suggestion
                    "Indicates the routing protocol. 'any' value can
                     be used to identify a limit that will apply for 
                     any active routing protocol.";
```

Reply:
I added 'any' to the common module. So, it should compile well on your side as well.